tools/unicodeCat.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const request = require('request');
+
+create(
+    'bmpLetters',
+    'https://raw.githubusercontent.com/mathiasbynens/unicode-6.2.0/' +
```

Comment:
Convinced, leave it as is. I erroneously assumed the tool would be run by the build process. Since it is only some sort of executable documentation for the unicode part of the parser, this is fine.